api/src/main/webapp/WEB-INF/cron_test.yaml

```diff
@@ -3,5 +3,7 @@ cron:
   schedule: every 24 hours
 - url: /v1/cron/checkFreeTierBillingUsageCloudTask
   schedule: every 24 hours
+- url: /v1/cron/checkFreeTierBillingUsage
```

Comment:
why not just leave it out? @NehaBroad

Reply:
That will interfere withe the process. Also just having checkFreeTierBillingUsageCloudTask will give us the confidence we need to push cloud task cron to PROD.